recipe-client-addon/lib/NormandyDriver.jsm

```diff
@@ -31,8 +31,11 @@ this.NormandyDriver = function(sandboxManager) {
   }
   const {sandbox} = sandboxManager;
 
+  Storage.seedDurability(sandbox);
```

Comment:
@andymikulski and I chatted in Vidyo about this. To summarize for the record, the main confusion is that when I said

> this will count only the first recipe run the first time as non-durable

What I was saying is that when storage has not proven itself to be durable, with the current code the first recipe to execute would see storage as not-proven-durable correctly, and all the other executed recipes would incorrectly see storage as durable.

The self-repair check looking for 2 was an implementation detail relating to how it handled retrieving and incrementing the number. It would have a value of 1 the first time, and never have a value of 0, so only a value of 2 would represent durability to Firefox restarting.

We also cleared up that sandboxes are annoying and confusing and complicate this a bit.